Makefile

```diff
@@ -146,6 +146,11 @@ cluster-sync-handler: cluster-sync-resources $(local_handler_manifest)
 	$(KUBECTL) apply -f deploy/crds/nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml
 	$(KUBECTL) delete --ignore-not-found -f $(local_handler_manifest)
 	$(KUBECTL) create -f $(local_handler_manifest)
+	until [[ \
+		$$($(KUBECTL) get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.desiredNumberScheduled}') \
+		-eq \
+		$$($(KUBECTL) get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.numberAvailable}') \
```

Comment:
Limite the number of checks we do by time or by cardinality, we don't want an infinite loop here.